src/set_theory/ordinal/arithmetic.lean

```diff
@@ -653,7 +653,7 @@ instance has_le.le.mul_covariant_class : covariant_class ordinal.{u} ordinal.{u}
 end⟩
 
 instance has_le.le.mul_swap_covariant_class :
-  covariant_class ordinal.{u} ordinal.{u} (function.swap (*)) (≤) :=
+  covariant_class ordinal.{u} ordinal.{u} (swap (*)) (≤) :=
```

Comment:
Function.swap and swap are two different functions. This might not be a no-op.

Or am I thinking of `flip`?

Reply:
As a mouse over reveals, these are in fact the same.